### PR TITLE
Add and update channels

### LINKED ISSUE #8250: Add: Republic of Costa Rica Legislative Assembly
### Channel ID

TVLegislativa.cr

### Stream URL

https://www.youtube.com/user/parlamentocr/live

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8348: Add: Channel 99
### Channel ID

CityTV.us

### Stream URL

https://edge-f.swagit.com/live/carlsbadca/live-3-a-1/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8421: Add: Fairfield Government Access – Channel 26
### Channel ID

CityofFairfieldChannel26.us

### Stream URL

http://204.17.179.200/CablecastPublicSite/?channel=1

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8422: Add: KFON TV
### Channel ID

KFONTV.us

### Stream URL

https://watchkfon.fontana.org/CablecastPublicSite/?channel=1

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8442: Add: Chuzhou News HD Channel
### Channel ID

ChuzhouNewsChannel.cn

### Stream URL

http://live.cztv.cc:85/live/xwpd.m3u8

### Notes

_No response_

### Please confirm the following

- [x] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8443: Add: Chuzhou Public Channel
### Channel ID

ChuzhouPublicChannel.cn

### Stream URL

http://live.cztv.cc:85/live/ggpd.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8444: Add: Chuzhou Science and Education Channel
### Channel ID

ChuzhouScienceEducationChannel.cn

### Stream URL

http://live.cztv.cc:85/live/sjpd.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8447: Add: GardenGroveTV3
### Channel ID

GGTV3.us

### Stream URL

GardenGroveTV3

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8454: Add: KRSU-TV
### Channel ID

KRSUDT1.us

### Stream URL

_No response_

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8461: Add: Glendale TV
### Channel ID

GlendaleTV.us

### Stream URL

https://reflect-gtv6-glendale.cablecast.tv/live-2/live/WIFI-1296k-540p/WIFI-1296k-540p.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8514: Add: 7 Televisión Región de Murcia [ES]
### Channel ID

7TelevisionRegiondeMurcia.es

### Stream URL

https://obelix.nopiensomas.com/stream/abr/first.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8515: Add: El Toro TV [ES]
### Channel ID

ElToroTV.es

### Stream URL

https://streaming003.gestec-video.com/hls/eltoro_bal.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8568: Add: 1TV Persian
### Channel ID

1TVPersian.us

### Stream URL

https://stream.persiantv1.com/ptv1/playlist1/index.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8587: Add: KPAS
### Channel ID

KPAS.us

### Stream URL

https://pasadenamed.secure.footprint.net/egress/bhandler/pasadenamed/streama/chunklist_b1192000.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8589: Add: KALO TV38
### Channel ID

KALOTV.us

### Stream URL

https://10380e91fda5e303.mediapackage.us-west-2.amazonaws.com/out/v1/e8a622ce6ed34d07b468dd6ea2f94ee8/index.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8591: Add: WLFG (Living Faith Television)
### Channel ID

WLFGDT1.us

### Stream URL

https://cdn-unified-hls.streamspot.com/ingest1/0b5c0f18e9/playlist.m3u8?origin=1

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8592: Add: KSCE 38.1 (LIFE! Christian)
### Channel ID

KSCEDT1.us

### Stream URL

https://2-fss-2.streamhoster.com/pl_138/200226-1359204-1/playlist.m3u8

### Notes

Channel 38.1 LIFE! Christian Variety programming
Channel 38.2 Vida Cristiana Spanish programming

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8593: Add: KSCE 38.2 (Vida Cristiana)
### Channel ID

KSCEDT2.us

### Stream URL

https://2-fss-2.streamhoster.com/pl_120/200226-1449024-1/playlist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8594: Add: KSCE 38.3 (LIFE! End Times TV)
### Channel ID

KSCEDT3.us

### Stream URL

https://2-fss-1.streamhoster.com/pl_122/200226-1427780-1/playlist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8595: Add: I-Cable News
### Channel ID

ICableNewsChannel.hk

### Stream URL

https://uc6.i-cable.com/live_freedirect/freehd209_h.live/playlist.m3u8

### Notes

HK's first 24-hour news channel. 

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8599: Add: Advocate Broadcasting Network
### Channel ID

AdvocateBroadcastingNetwork.ng

### Stream URL

https://www.lyngsat-stream.com/tvchannels/ng/Advocate-Broadcasting-Network.html

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8600: Add: Channels TV
### Channel ID

ChannelsTV.ng

### Stream URL

https://www.youtube.com/watch?v=-c8LoR84Xjs&ab_channel=ChannelsTelevision

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8601: Add: Galaxy TV
### Channel ID

GalaxyTV.ng

### Stream URL

https://www.galaxytvonline.com/video?live=livestream

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8602: Add: Silverbird TV
### Channel ID

SilverbirdTV.ng

### Stream URL

https://silverbirdtv.com/live/

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8603: Add: TVC Entertainment
### Channel ID

TVC.ng

### Stream URL

https://www.tvcentertainment.tv/livestream/

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8605: Add: Wazobia Max TV Nigeria
### Channel ID

WazobiaMaxTVNigeria.ng

### Stream URL

https://www.wazobiamax.ng/

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8607: Add: Wap TV
### Channel ID

WapTV.ng

### Stream URL

https://waptvchannel.com/live-tv-2/

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8614: Add: Trust TV
### Channel ID

TrustTV.ng

### Stream URL

https://trusttv.com/

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8620: Add: TCT HD [24/7] | (US)
### Channel ID

TCTHD.us

### Stream URL

https://livestreamapis.com/v3/accounts/29565692/events/9276365/master.m3u8?token=9bfa600cf944aae6abaf595ca9051502&clientId=18436&timestamp=1662762442478

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8621: Add: TCT KIDS [24/7] | (US)
### Channel ID

TCTKids.us

### Stream URL

https://livestreamapis.com/v3/accounts/29565692/events/9321713/master.m3u8?token=163b3c3c1c23fcc3d2517b11e83d723a&clientId=18436&timestamp=1662762649658

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8622: Add: Positiv
### Channel ID

PositivTV.us

### Stream URL

https://tbn-jw.cdn.vustreams.com:443/live/positivtv/live.isml/live-audio_1=128000-video=2800000.m3u8 / https://tbn-jw.cdn.vustreams.com/live/positivtv/live.isml/952dda4a-bfff-407c-803d-000271d853b4.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8625: Add: Arroyo
### Channel ID

Arroyo.us

### Stream URL

https://pasadenamed.secure.footprint.net/egress/bhandler/pasadenamed/streamb/chunklist_b1192000.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8626: Add: PCCtv
### Channel ID

PCCtv.us

### Stream URL

https://pasadenamed.secure.footprint.net/egress/bhandler/pasadenamed/streamc/chunklist_b1192000.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8627: Add: Channel 26
### Channel ID

Pinole26.us

### Stream URL

https://cdn3.wowza.com/5/cXdyRHF0Z3kxN0k2/pinole/G0032_004/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8628: Add: Channel 28
### Channel ID

Pinole28.us

### Stream URL

https://cdn3.wowza.com/5/cXdyRHF0Z3kxN0k2/pinole/G0032_002/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8629: Add: City of Placentia
### Channel ID

PlacentiaTV.us

### Stream URL

https://cdn3.wowza.com/5/M0lyamVmM2JWcjhQ/placentia/G0928_002/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8630: Add: America Tucuman
### Channel ID

AmericaTucuman.ar

### Stream URL

https://youtu.be/UcQRYiWPJtY

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8634: Add: BVC
### Channel ID

BVC.ar

### Stream URL

https://vivo.solumedia.com:19360/bvconline/bvconline.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8635: Add: Cadena V!VO
### Channel ID

CadenaVivo.ar

### Stream URL

https://videostream.shockmedia.com.ar:19360/tvnoficial/tvnoficial.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8638: Add: VTN / Victory Television Network
### Channel ID

KVTNDT1.us

### Stream URL

https://api.new.livestream.com/accounts/15669040/events/4554297/broadcasts/232863013.secure.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8640: Add: Retro TV East | (US)
### Channel ID

RetroTVEast.us

### Stream URL

https://bcovlive-a.akamaihd.net/5e531be3ed6c41229b2af2d9bffba88d/us-east-1/6183977686001/playlist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8641: Add: KBS Korea
### Channel ID

KBSKorea.kr

### Stream URL

https://m.youtube.com/watch?v=HnfpTMtfFk8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8659: Add: Canal 10 TV Río Negro
### Channel ID

Canal10TVRioNegro.ar

### Stream URL

https://panel.dattalive.com/8204/8204/chunklist_w258528815.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8664: Add: Riverside TV
### Channel ID

RiversideTV.us

### Stream URL

https://cdn3.wowza.com/5/RXJNMFI3VlVkOEFP/riversideca/G0879_001/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8667: Add: SD CityTV
### Channel ID

CityTVSanDiego.us

### Stream URL

https://cdn3.wowza.com/5/dk84U1p2UUdoMGxT/sandiego/G1826_005/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8668: Add: SF Commons 29
### Channel ID

SFCommons29.us

### Stream URL

https://reflect-bayarea.cablecast.tv/live-5/live/WIFI-1096k-360p/WIFI-1096k-360p.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8669: Add: SF Commons 76
### Channel ID

SFCommons76.us

### Stream URL

https://reflect-bayarea.cablecast.tv/live-5/live/WIFI-1096k-360p/WIFI-1096k-360p.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8670: Add: The Government Channel - Channel 26
### Channel ID

SFGovTV.us

### Stream URL

https://cdn3.wowza.com/5/V2Y2VmhqMEFDTUkx/sanfrancisco/G0051_003/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8671: Add: The City Channel - Channel 78
### Channel ID

SFGovTV2.us

### Stream URL

https://cdn3.wowza.com/5/V2Y2VmhqMEFDTUkx/sanfrancisco/G0051_008/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8673: Add: 3ABN Praise Him Music Network
### Channel ID

3ABNPraiseHimMusicNetwork.us

### Stream URL

https://3abn-live.akamaized.net/hls/live/2010551/Praise_Him/SQs/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8674: Add: 3ABN Russia
### Channel ID

3ABNRussia.ru

### Stream URL

https://3abn-live.akamaized.net/hls/live/2010548/Russia/HQs/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8675: Add: ABP News USA
### Channel ID

ABPNews.in

### Stream URL

https://abplivetv.akamaized.net/hls/live/2043010/hindi/master_720p.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #8685: Add: La Nación+
### Channel ID

LaNacionPlusHD.ar

### Stream URL

http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/watch?v=HlhZ0XZHHaw

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### PR DESCRIPTION
**Pull request by Alstruit**
Closes the following:
Fixes #8421, fixes #8422, fixes #8442, fixes #8443, fixes #8444, fixes #8454, fixes #8461, fixes #8514, fixes #8515, fixes #8568, fixes #8587, fixes #8589, fixes #8591, fixes #8592, fixes #8593, fixes #8594.
Fixes #8595, fixes #8601, fixes #8603, fixes #8605, fixes #8607, fixes #8620, fixes #8621, fixes #8622, fixes #8625, fixes #8626, fixes #8627, fixes #8628, fixes #8629, fixes #8634, fixes #8635, fixes #8638, fixes #8640, fixes #8659, fixes #8664, fixes #8667, fixes #8668, fixes #8669, fixes #8670, fixes #8671, fixes #8673, fixes #8674, fixes #8675.
Fixes #8250, fixes #8348, fixes #8447, fixes #8599, fixes #8600, fixes #8602, fixes #8614, fixes #8630, fixes #8641, fixes #8685.

iptv-org/database#399 needs to be merged first to apply ID fixes.

### CODE REVIEW

File: streams/ru.m3u [hunk 1]
```diff
@@ -727,3 +727,5 @@ http://ott-cdn.ucom.am/s20/index.m3u8
 http://185.97.150.19:8082/2401
 #EXTINF:-1 tvg-id="" status="blocked",ЯМАЛ 1
 https://bl.rutube.ru/livestream/eff3ed0c2cdde88e8c5fdc515aee2c9f/index.m3u8?e=1653312522&s=rAfTRV43vtwOtE12zJ8UQQ&scheme=https
+#EXTINF:-1 tvg-id="" status="",Три Ангела (720p)
```
Comment (freearhey) [inline streams/ru.m3u:730]:
```suggestion
#EXTINF:-1 tvg-id="3ABNRussia.ru" status="",Три Ангела (720p)
```

https://iptv-org.github.io/?q=%D0%A2%D1%80%D0%B8%20%D0%90%D0%BD%D0%B3%D0%B5%D0%BB%D0%B0

File: streams/ng.m3u [hunk 2]
```diff
@@ -7,3 +7,23 @@ http://185.105.4.193:1935/ltv/myStream/playlist.m3u8
...
+https://wazobia.live:8333/channel/wmaxph.m3u8
+#EXTINF:-1 tvg-id="WapTV.ng" status="",Wap TV (720p)
+https://newproxy3.vidivu.tv/waptv/index.m3u8
+#EXTINF:-1 tvg-id="TrustTV.ng" status="",Trust TV (720p)
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCTlqstA2Wrt4fimd_VWKr8g/live
+#EXTINF:-1 tvg-id="SilverbirdTV.ng" status="",Silverbird TV (480p)
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://dai.ly/x81uq2d
+#EXTINF:-1 tvg-id="ChannelsTV.ng" status="",Channels TV (720p)
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/c/ChannelsTelevision/live
+#EXTINF:-1 tvg-id="" status="",ABN Advocate Broadcasting Network (720p)
```
Comment (freearhey) [inline streams/ng.m3u:28]:
```suggestion
#EXTINF:-1 tvg-id="AdvocateBroadcastingNetwork.ng" status="",ABN Advocate Broadcasting Network (720p)
```

https://iptv-org.github.io/?q=Advocate%20Broadcasting%20Network

File: streams/ng.m3u [hunk 3]
```diff
@@ -7,3 +7,23 @@ http://185.105.4.193:1935/ltv/myStream/playlist.m3u8
 https://api.visionip.tv/live/ASHTTP/visiontvuk-entertainment-ntai-hsslive-25f-4x3-MB/playlist.m3u8
 #EXTINF:-1 tvg-id="TVCNews.ng" status="error",TVC News
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/c/tvcnewsnigeria/live
+#EXTINF:-1 tvg-id="GalaxyTV.ng" status="",Galaxy TV (576p)
+https://5d846bfda90fc.streamlock.net:1935/live/galaxytv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="",TVC (1080p)
```
Comment (freearhey) [inline streams/ng.m3u:12]:
```suggestion
#EXTINF:-1 tvg-id="TVC.ng" status="",TVC (1080p)
```

https://iptv-org.github.io/?q=TVC%20country:NG

**Review by freearhey — CHANGES_REQUESTED** [review]
Here are some missing IDs. 

Other than that, everything is great.

**Review by freearhey — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Comment by LaneSh4d0w** [comment]
Awesome job @Alstruit , now time for the big merge.

**Pull request merged**